proto/pdpb.proto

```diff
@@ -295,3 +295,35 @@ message Error {
     optional BootstrappedError     bootstrapped = 2;
     optional StoreIsTombstoneError is_tombstone = 3;
 }
+
+service PlacementService {
```

Comment:
Isn't that a little abstract? :thinking: 

BTW, I think we should omit `Service`.